tests/integration/test_hedged_requests_parallel/test.py

```diff
@@ -0,0 +1,158 @@
+import os
+import sys
+import time
+
+import pytest
+
+sys.path.insert(0, os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+from helpers.cluster import ClickHouseCluster
+from helpers.network import PartitionManager
+
+cluster = ClickHouseCluster(__file__)
+
+NODES = {'node_' + str(i): None for i in (1, 2, 3, 4)}
+NODES['node'] = None
+
+sleep_time = 30
+
+@pytest.fixture(scope="module")
+def started_cluster():
+    cluster = ClickHouseCluster(__file__)
+    NODES['node'] = cluster.add_instance(
+    'node', with_zookeeper=True, stay_alive=True, main_configs=['configs/remote_servers.xml'], user_configs=['configs/users.xml'])
+
+    for name in NODES:
+        if name != 'node':
+            NODES[name] = cluster.add_instance(name, with_zookeeper=True, user_configs=['configs/users1.xml'])
```

Comment:
Looks like the test does not really requires zookeeper and ReplicatedMergeTree since it uses remote_servers, right?

Reply:
Yes, you are right, it would be good to remove zookeeper and ReplicatedMergeTree from this test, I will do it, thanks.